src/.env.sample

```
@@ -0,0 +1,259 @@
+
+#maximum number of tasks allowed in a project
+MAX_PROJECT_TASK_COUNT= 10
+
+#consumption portal name ex: self/sunbird/elevate-project . Any value other than self will give a combined result of self resources and curresponding service
+CONSUMPTION_SERVICE=self
+
+#kafka topic for publish
+PROJECT_PUBLISH_KAFKA_TOPIC=publishtopic
+
+#maximum body length of file, it can be Infinity or length in bytes
+MAX_BODY_LENGTH_FOR_UPLOAD=Infinity/5242880 #5MB in bytes
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)